ext/opcache/zend_shared_alloc.c

```diff
@@ -40,7 +33,7 @@
 # include <stdio.h>
 #endif
 
-#ifdef HAVE_MPROTECT
+#if defined(HAVE_MPROTECT) || defined(HAVE_MEMFD_CREATE)
```

Comment:
I think it was because we did not want to [enable memfd_create for FreeBSD](https://github.com/php/php-src/pull/16570/files#diff-8838190023fd4fbad393e08e4dddbac04ed7b1a37276be671d45076cf21086dfR85).